src/main/kotlin/no/nav/syfo/infrastructure/kafka/KandidatStatusProducer.kt

```diff
@@ -41,14 +45,16 @@ data class KandidatStatusRecord(
     val uuid: UUID,
     val createdAt: OffsetDateTime,
     val personident: String,
+    val veilederident: String?,
     val status: StatusDTO,
 ) {
     companion object {
-        fun fromSenOppfolgingKandidat(kandidat: SenOppfolgingKandidat): KandidatStatusRecord =
+        fun fromSenOppfolgingKandidat(kandidat: SenOppfolgingKandidat, veilederident: String?): KandidatStatusRecord =
```

Comment:
Har bare utvidet med `veilederident` (som hentes fra vurderingen) her foreløpig. Status for en kandidat kan være `KANDIDAT` eller `FERDIGBEHANDLET`.

Reply:
Hmm, ja kanskje statusDTOen inneholdt et sånt felt 🤔